Library/Formula/lldpd.rb

```diff
@@ -21,17 +21,19 @@ class Lldpd < Formula
 
   def install
     readline = Formula["readline"]
-    args = ["--prefix=#{prefix}",
-            "--sysconfdir=#{etc}",
-            "--localstatedir=#{var}",
-            "--with-xml",
-            "--with-readline",
-            "--with-privsep-chroot=/var/empty",
-            "--with-privsep-user=nobody",
-            "--with-privsep-group=nogroup",
-            "--with-launchddaemonsdir=no",
-            "CPPFLAGS=-I#{readline.include} -DRONLY=1",
-            "LDFLAGS=-L#{readline.lib}"]
+    args = [
```

Comment:
It's a stylistic thing but personally prefer the use of `%W` with spaced arrays like this, and the consequent removing of the `""` around the arguments. Merging here though, no worries.